### PR TITLE
5806 - Input mobile enhancement puppeteer script

### LINKED ISSUE #5806: Input: Create a Puppeteer Script for Input's Mobile Enhancements and style changes
Describe the Issue
Create a unit testing for Input's Mobile Enhancements and style changes.

See attached BDD for more information.
[story_github5606.docx](https://github.com/infor-design/enterprise/files/7442092/story_github5606.docx)

Related github issue
#5606

### PR DESCRIPTION
**Pull request by janahintal**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Input puppeteer test for #5606

**Related github/jira issue (required)**:

Closes #5806
Related Issue #5606

**Steps necessary to review your pull request (required)**:

1. pull this branch
2. build and run the app
3. run `npm run e2e:puppeteer input`
4. It should pass.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

**Review by tjamesallen15 — CHANGES_REQUESTED** [review]
Looks good although we need to fix the lint issues. You can chat me if you have issues fixing it @janahintal I can help you with that.

File: test/jest-puppeteer.config.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ module.exports = {
   // },
   // https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteerlaunchoptions
   launch: {
-    headless: true,
+    headless: false,
+    slowMo: 30,
```
Comment (tmcconechy) [inline test/jest-puppeteer.config.js:9]: Should revert these....

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Requesting changes as we shouldnt sloMo the tedsts and make them not headless except for testing local

**Comment by tmcconechy** [comment]
any updates @janahintal ?

**Comment by janahintal** [comment]
> any updates @janahintal ?

Hi @tmcconechy sorry I was on leave last friday. Will fix this today. Thank you!

**Review by ericangeles — APPROVED** [review]
🍌

**Review by EdwardCoyle — APPROVED** [review]
Looks good, thanks @janahintal

**Pull request merged**